addons/html_editor/static/src/main/media/media_dialog/image_selector.js

```diff
@@ -51,7 +51,14 @@ export class AutoResizeImage extends Attachment {
 const newLocal = "img-fluid";
 export class ImageSelector extends FileSelector {
     static mediaSpecificClasses = ["img", newLocal, "o_we_custom_image"];
-    static mediaSpecificStyles = [];
+    static mediaSpecificStyles = [
```

Comment:
As you added the rounded corner option in `html_builder`, either this should be in the `html_builder` addon, either it should be in another PR as a fix if it concerns `html_editor` as well.